src/Host/Client/Impl/ExecutionTracing/RExecutionTracer.cs

```diff
@@ -171,18 +171,16 @@ internal sealed class RExecutionTracer : IRExecutionTracer {
 
         public Task<bool> StepOutAsync(CancellationToken cancellationToken = default(CancellationToken)) =>
             StepAsync(cancellationToken, "c", async inter => {
-                using (var eval = await Session.BeginEvaluationAsync(cancellationToken)) {
-                    try {
-                        // EvaluateAsync will push a new toplevel context on the context stack before
-                        // evaluating the expression, so tell browser_set_debug to skip 1 toplevel context
-                        // before locating the target context for step-out.
-                        await eval.ExecuteAsync("rtvs:::browser_set_debug(1, 1)", REvaluationKind.Normal);
-                    } catch (RException) {
-                        _stepTcs.TrySetResult(false);
-                        return false;
-                    }
-                    return true;
+                try {
+                    // EvaluateAsync will push a new toplevel context on the context stack before
+                    // evaluating the expression, so tell browser_set_debug to skip 1 toplevel context
+                    // before locating the target context for step-out.
+                    await Session.ExecuteAsync("rtvs:::browser_set_debug(1, 1)", REvaluationKind.Normal, cancellationToken);
```

Comment:
I would prefer this to be inside `BeginEvaluation` (or `BeginInteraction` if we're get rid of the former), to ensure that it always happens at the REPL prompt. Otherwise it might interact with things in an unpredictable way.